app/src/pages/review/[resourceType]/[id].tsx

```diff
@@ -0,0 +1,84 @@
+import { Prism } from '@mantine/prism';
+import { Center, Divider, Grid, Paper, Space, Tabs, Text } from '@mantine/core';
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
+import { FhirArtifact } from '@/util/types/fhir';
+import { ArtifactResourceType } from '@/util/types/fhir';
+import { useRouter } from 'next/router';
+import { trpc } from '@/util/trpc';
+
+/**
+ * Component which renders a page that displays the JSON data of a resource. Also will eventually
+ *  provide the user with the ability to make review comments and visualize previous review comments.
+ */
+export default function CommentPage({ jsonData }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  const jsonResourceType = jsonData.resourceType;
+  const router = useRouter();
+  const { resourceType: draftResourceType, id: draftID } = router.query;
+  const { authoring } = router.query;
+
+  const { data: resource } = trpc.draft.getDraftById.useQuery({
+    id: draftID as string,
+    resourceType: draftResourceType as ArtifactResourceType
+  });
```

Comment:
After chatting with Matt about this task, we decided that it would make more sense and would help us later on if we don't use `getServerSideProps` in this component and use `trpc` in order to fetch an artifact from the measure repository. This requires adding a procedure similar to this one in `service.ts` (this one is defined in `draft.ts`). Once you write a `getArtifactByid` procedure in `service.ts`, then you can use it here just like the above! So it will be something like the following:

```
if (authoring === true) {
     // we will want to fetch the artifact from the draft artifact db
     const { data: resource } = trpc.draft.getDraftById.useQuery...
} else {
    // authoring will be false, therefore we will want to fetch the artifact from the measure repository
    // we will want to do this with a trpc procedure that you will need to write in service.ts and then call it here
    const { data: resource } = trpc.service.getDraftById.useQuery...
}
```

Does that make sense? Happy to help with the `trpc procedures` definitions :)